server.py

```diff
@@ -69,6 +69,7 @@ def validate():
     })
 
     s = Schema({
+        Optional('heartbeat'): bool,
```

Comment:
I think it's fine as a true boolean. Keeps away from the ambiguity over what is true and false.